src/css/style.css

```diff
@@ -210,8 +210,13 @@ footer {
   }
 }
 
-/* @screen xl {
-  .socials a {
+@media (min-width: 1920px) {
+  /* .socials a {
     @apply mx-16;
+  } */
+
+  .event-card {
+    height: 575px;
+    width: 570px;
   }
-} */
+}
```

Comment:
I'll have to look into this. I added this because the event cards keep growing and get a really big past 1920x1080. There is likely an alternative way to fix this that won't break it.

Reply:
Okay, so I went ahead and removed it. The event cards now stay at the same size for larger screen sizes. I'll let you guys decide if this is how you want it to look. In addition to this I changed a few styles for the Connect logo below it so that they are aligned better and have a little more space between each other. It looks less cluttered.